### ISSUE TITLE
I need some help with tensorflow.org/js object recognition

### ISSUE THREAD

**Issue by mtrucc** [chansoopark98/NodeJS-based-DeepLearning#1]
Here is a working sample code

https://github.com/mtrucc/react-ts-fpkqes

Here is the working model

https://github.com/mtrucc/react-ts-fpkqes/tree/main/public/mask_model

This is the model I trained myself and converted using the command, it doesn't work

https://github.com/mtrucc/react-ts-fpkqes/tree/main/public/mask_model2

You can switch models by modifying the path in the code

https://github.com/mtrucc/react-ts-fpkqes/blob/6cb2ce09f4c44673098749041244b0f998f8d8f4/src/components/mask.js#L35

Install tensorflowjs

```
!pip install tensorflowjs
```

Return

```
Successfully installed flatbuffers-1.12 gast-0.4.0 keras-2.9.0 packaging-20.9 protobuf-3.20.0 tensorboard-2.9.0 tensorflow-2.9.0 tensorflow-estimator-2.9.0 tensorflowjs-3.19.0
```

Here is my conversion command in colab

```
tensorflowjs_converter --input_format=tf_saved_model \
                        --output_format=tfjs_graph_model \
  /content/gdrive/MyDrive/customTF2/data/inference_graph/saved_model \
  /content/gdrive/MyDrive/customTF2/data/inference_graph/saved_model_js
I don't know where is the problem? If you need more details, I can add.
```

I don't know where is the problem? If you need more details, I can add.

**Comment by chansoopark98** [comment]
Hello, can you elaborate on which part is not working?
I'd appreciate it if you could attach an error code or a picture

**Comment by mtrucc** [comment]
The code I found via
`const model = await tf.loadGraphModel('assets/converted_tfjs/model.json');`
I'm very sorry, because I don't know how you trained the model. At present, I have an error when loading the model. I want to ask how you generated the model.

**Comment by chansoopark98** [comment]
Ah, I see.
In the case of tf.loadGraphModel, a static graph type model is loaded.
https://github.com/chansoopark98/Tensorflow-Keras-Object-Detection/blob/main/convert_frozen_graph.py
Please refer to my repository.
The method is to build a model trained with TensorFlow or Keras, load and apply weights.
Running that code creates a frozen graph with the specified path.

When the conversion is complete, convert it to the tensorflowjs_converter ./checkpoints/new_tfjs_frozen/frozen_graph.pb ./checkpoints/converted_tfjs/ --input_format=tf_frozen_model --output_node_names='Identity' --quantize_float16 command.

Once the transformation is complete, you can load the model from tfjs using tf.loadGraphModel.
Please try it and contact us if any errors occur.

**Comment by chansoopark98** [comment]
I closed the issue by mistake.
Please try the method that I told you before replying!
Have a nice day

2022년 8월 8일 (월) 오후 12:51, mtrucc ***@***.***>님이 작성:

**Comment by mtrucc** [comment]
After research, I found that our models are inconsistent
I use save model
As noted https://github.com/tensorflow/models/issues/8858#issuecomment-658348127 frozen graphs are not supported in TF2.
I did not transform the model, I reanalyzed the output data and wrote a new js function to process it
